Source/WebGPU/WebGPU/BindGroup.mm

```diff
@@ -78,13 +78,280 @@ static MTLRenderStages metalRenderStage(ShaderStage shaderStage)
         simd::make_float3(+1.79968f, -0.53503f, +0.00012f),
         simd::make_float3(-0.97284f, 0.30145f, -1.13348f));
 }
+
+static MTLPixelFormat metalPixelFormat(CVPixelBufferRef pixelBuffer, size_t plane)
+{
+    auto pixelFormat = CVPixelBufferGetPixelFormatType(pixelBuffer);
+    auto biplanarFormat = ^(int plane) {
+        return plane ? MTLPixelFormatRG8Unorm : MTLPixelFormatR8Unorm;
+    };
+
+    switch (pixelFormat) {
```

Comment:
I'm going to assume that this big table is right

Or at least, if it isn't, rigorous testing will discover any incorrect parts